conda_build/post.py

```diff
@@ -320,7 +320,7 @@ def mk_relative(m, f):
     if not is_obj(path):
         return
 
-    if sys.platform.startswith('linux'):
+    if sys.platform.startswith('linux') or ('bsd' in sys.platform):
```

Comment:
@NewbiZ If you wouldn't break things and would write a good code from the beginning, I would keep silence.

P.S. You are welcome.